ui/design-system-components/src/components/ProfileEngagements/Engagement/Following.tsx

```diff
@@ -3,26 +3,25 @@ import Spinner from '@akashaorg/design-system-core/lib/components/Spinner';
 import Stack from '@akashaorg/design-system-core/lib/components/Stack';
 import Entry from '../Entry';
 import EmptyEntry from '../Entry/EmptyEntry';
-import { tw } from '@twind/core';
 import { useIntersection } from 'react-use';
 import { getColorClasses } from '@akashaorg/design-system-core/lib/utils';
 import { AkashaFollowing } from '@akashaorg/typings/lib/ui';
 import { EngagementProps } from '../types';
+import { InfoCardProps } from '@akashaorg/design-system-core/lib/components/InfoCard';
 
 export type FollowingProps = {
   following: AkashaFollowing;
-  ownerUserName: string;
-  viewerIsOwner: boolean;
-} & EngagementProps;
+} & EngagementProps &
+  Pick<InfoCardProps, 'titleLabel' | 'bodyLabel'>;
 
 const Following: React.FC<FollowingProps> = ({
   authenticatedDID,
   followList,
   following,
   profileAnchorLink,
-  ownerUserName,
-  viewerIsOwner,
   loadMore,
+  titleLabel,
```

Comment:
Nice refactoring here but the prop naming is confusing. It's hard to tell what they are referring to unless you look at the implementation for the component.  As a suggestion you can rename it and the `bodyLabel` as, `emptyEntryTitleLabel` and `emptyEntryBodyLabel` respectively.